_data/changelogs/docs-showcase.yml

```diff
@@ -0,0 +1,10 @@
+title: Showcase
+type: documentation
+changelogURL:
+items:
+  - date: NNNN-NN-NN
+    summary: Remove broken links that have no alternate for redirects. 
+    summaryAdditional: Correct misspelling that Siteimprove noted.
+    affectsGuidance: true
+    githubPr: 2586
+    githubRepo: uswds-site
```

Comment:
## Created changelog

After discussing with the team, we decided to add a changelog and keep track of larger changes such as addition and removal of links.